Makefile

```diff
@@ -31,7 +31,7 @@ static-fmt:
 
 # lint our toml files
 static-toml:
-	cargo install taplo-cli@0.9.0
+	cargo install taplo-cli@0.9.0 --locked
```

Comment:
Needs to be using `--locked` now, otherwise it pulls in transitive dependencies that depend on Rust 1.82